app/controllers/conversations.js

```diff
@@ -48,32 +50,70 @@ class ConversationsController extends BaseController {
         $or: [
           {
             type: "u",
-            owner_id: ObjectId(this.sessionRepository.getSessionUserId(ws)),
+            owner_id: ObjectId(currentUserId),
             opponent_id: conversationParams.opponent_id,
           },
           {
             type: "u",
             owner_id: ObjectId(conversationParams.opponent_id),
-            opponent_id: this.sessionRepository.getSessionUserId(ws),
+            opponent_id: currentUserId,
           },
         ],
       });
-      if (existingConversation)
+
+      if (existingConversation) {
+        const existingParticipants = await ConversationParticipant.findAll({
+          conversation_id: existingConversation._id,
+        });
+        if (existingParticipants.length !== 2) {
+          const existingParticipantsArray = existingParticipants.map((u) =>
+            u.user_id.toString()
+          );
+          const userId = [...participants, conversationParams.owner_id]
+            .filter(
+              (uId) => !existingParticipantsArray.includes(uId.toString())
+            )
+            .map((u) => u.toString())[0];
+
+          const participant = new ConversationParticipant({
+            user_id: ObjectId(userId),
+            conversation_id: existingConversation._id,
+          });
+          await participant.save();
+
+          await PacketProcessor.deliverToUserOrUsers(
+            ws,
+            {
+              conversation_create: {
```

Comment:
let's call it `event_conversation_create`
and pass the complete `conversation` body, similar to response for Create Conversation API